controller/eks-cluster-config-handler_test.go

```diff
@@ -3,11 +3,12 @@ package controller
 import (
 	"errors"
 
-	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go-v2/aws"
 
-	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation"
+	cftypes "github.com/aws/aws-sdk-go-v2/service/cloudformation/types"
 
-	awssdkeks "github.com/aws/aws-sdk-go/service/eks"
```

Comment:
I believe the old alias was easier to follow compared to current change

Reply:
added back previous alias